ospfd/ospf_vty.c

```diff
@@ -10118,6 +10118,21 @@ static int ospf_print_vty_helper_dis_rtr_walkcb(struct hash_bucket *bucket,
 	return HASHWALK_CONTINUE;
 }
 
+static int ospf_print_json_helper_enabled_rtr_walkcb(struct hash_bucket *bucket,
+						 void *arg)
+{
+	struct advRtr *rtr = bucket->data;
+	struct json_object *json_rid_array = (struct json_object *)arg;
```

Comment:
typcasting is not needed: `struct json_object *json_rid_array = arg;`

Reply:
Addressed @ton31337 , thanks.